src/mono/wasm/Wasm.Build.Tests/WorkloadTests.cs

```diff
@@ -58,13 +59,20 @@ public void FilesInUnixFilesPermissionsXmlExist()
             // have the unixFilePermissions for that
             // Expect just the emscripten ones here for now
 
-            // linux doesn't have Emscripten.Python package, so only 2 there
             int expectedPermFileCount = RuntimeInformation.IsOSPlatform(OSPlatform.Linux) ? 4 : 5;
 
-            int permFileCount = unixPermFiles.Count();
-            if (permFileCount != expectedPermFileCount)
-                throw new XunitException($"Expected to find {expectedPermFileCount} UnixFilePermissions.xml files, but got {permFileCount}."
-                                            + $"{Environment.NewLine}Files: {string.Join(", ", unixPermFiles)}");
+            // extract pack names from ./artifacts/bin/dotnet-latest/packs/Microsoft.NET.Runtime.Emscripten.3.1.34.Sdk.linux-x64/8.0.0-rtm.23470.1/data/UnixFilePermissions.xml
+            var packNames = unixPermFiles.Select(f => Path.GetFileName(Path.GetDirectoryName(Path.GetDirectoryName(Path.GetDirectoryName(f))))).ToArray();
+
+            Assert.Contains(packNames, name => name!.StartsWith("Microsoft.NET.Runtime.Emscripten.3.1.34.Cache."));
+            Assert.Contains(packNames, name => name!.StartsWith("Microsoft.NET.Runtime.Emscripten.3.1.34.Node."));
+            Assert.Contains(packNames, name => name!.StartsWith("Microsoft.NET.Runtime.Emscripten.3.1.34.Sdk."));
```

Comment:
should leave the version out

Reply:
Um could be useful when we have more than one workload installed with different emsdk versions. but yeah, not a big deal, because to have that scenario always test correctly, we'll need to match up other versions.
I'll fix.

Reply:
we can match to the names from the flow, I just want to avoid having to tweak this to upgrade.  it can be fixed for 8.0, 7.0, 6.0